EmailValidator/Validation/MultipleValidationWithAnd.php

```diff
@@ -3,23 +3,49 @@
 namespace Egulias\EmailValidator\Validation;
 
 use Egulias\EmailValidator\EmailLexer;
+use Egulias\EmailValidator\Exception\InvalidEmail;
 use Egulias\EmailValidator\Validation\Exception\EmptyValidationList;
 
 class MultipleValidationWithAnd implements EmailValidation
 {
+    /**
+     * @var EmailValidation[]
+     */
     private $validations = [];
+
+    /**
+     * @var array
+     */
     private $warnings = [];
+
+    /**
+     * @var InvalidEmail
+     */
     private $error;
-    
-    public function __construct(array $validations)
+
+    /**
+     * @var bool
+     */
+    private $breakIfError;
+
+    /**
+     * @param EmailValidation[] $validations  The validations.
+     * @param bool              $breakIfError If true, it breaks out of validation loop when error occurs,
+     *                                        it means returned MultipleErrors might not contain all causes of errors. (false by default)
```

Comment:
`it means returned MultipleErrors will contain a single error, the first found.`